source/api-docs.txt

```diff
@@ -31,13 +31,21 @@ you access the application on (either ``8278``, ``8080``, or ``443``).
 How the REST API Works
 ----------------------
 
-The Relational Migrator REST API:
+The Relational Migrator REST API runs on the client computer or server
+on which you install Relational Migrator. By default, Relational
```

Comment:
[suggestion]

Consider: "...runs on the client computer or server that has Relational Migrator installed."

Reply:
Done!